var/spack/repos/builtin/packages/podio/package.py

```diff
@@ -112,16 +126,20 @@ class Podio(CMakePackage):
         description="Use the specified C++ standard when building.",
     )
     variant("sio", default=False, description="Build the SIO I/O backend")
+    variant("rntuple", default=False, description="Build the RNTuple backend")
```

Comment:
I'm not completely sure but this needs ROOT 6.28.04. I remember I started with 6.28.00 or around there but there was some change in some interface around .04 that I implemented